src/debugger/ios/deviceDeployer.ts

```diff
@@ -23,7 +23,7 @@ export class DeviceDeployer {
                 "Build", "Products", "Debug-iphoneos", `${projectName}.app`);
             return new CommandExecutor(this.projectRoot)
                 .spawn("ideviceinstaller", ["-i", pathToCompiledApp]).catch((err) => {
-                    if ((<any>err).code === "ENOENT") {
+                    if ((<any>err).errorCode === 101 && (<any>err).innerError.code === "ENOENT") {

```

Comment:
Instead of the magic number 101, you should use `InternalErrorCode.CommandFailed`